internal/discord/discord.go

```diff
@@ -7,8 +7,9 @@ import (
 )
 
 type Discord struct {
-	Token   string
-	Session *discordgo.Session
+	Token           string
+	Session         *discordgo.Session
+	ModLogChannelID string
```

Comment:
we dont have to do this now, but there may be a chance in the future we are implementing several logging channels for different things

Reply:
(resolving btw no action required)